src/services/search/SearchService.res

```diff
@@ -0,0 +1,17 @@
+open UrlType;
+
+let make = (token: string) => {
```

Comment:
Doesn't have to be a component right? Also I would say we can bundle all interaction with the Spotify API in a single SpotifyService, shouldn't be too many endpoints we have to call

Reply:
I can create a combined service on top of it but would keep it separated on granular level

Reply:
And make is not only used for react components, but also for init functions like this one (See belt array)